lambdas/get-addresses/tests/unit/app.test.ts

```diff
@@ -0,0 +1,72 @@
+import { APIGatewayProxyEvent } from "aws-lambda";
```

Comment:
The [GDS Way for Node.js](https://gds-way.cloudapps.digital/manuals/programming-languages/nodejs/#project-directory-structure) says:

 > Store test files within the implementation

This is based on the standard JS way of working which doesn't necessarily fit with Typescript.

Thoughts on co-locating of tests vs a separate unit test folder?